src/zookeeper/deploy/refresh.sh

```diff
@@ -17,5 +17,7 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+pushd $(dirname "$0") > /dev/null
 
```

Comment:
For this empty refresh.sh, we can add a logic in pai service management that if file not exists, just ignore it. If so, not the all services has to add an empty refresh.sh if they are nothing to do with it.

Reply:
I think it's ok to let it be, for it's convinent to add more things to do of refresh in the future?